src/project-tests/calculator-tests.js

```diff
@@ -51,10 +51,11 @@ export default function createCalculatorTests() {
       it(`${reqNum}. My calculator should contain a clickable element
       containing an "=" (equal sign) with a corresponding id="equals"`,
       function() {
-        assert.isNotNull(document.getElementById(_eq));
+        let element = document.getElementById(_eq);
+        assert.isNotNull(element);
 
         assert.strictEqual(
-          document.getElementById(_eq).innerHTML,
+          element.localName === 'input' ? element.value : element.innerText,
```

Comment:
Could you please change this to use `element.nodeName`?

`element.localName` is supposed to only work for XML docs. `element.tagName` looks a little better, but this Stackoverflow question has some comments explaining that `element.nodeName` is supported by the most browsers and is more standard. `element.nodeName` is supposed to return upper case, so you should also normalize it to always be lowercase, so you can compare with a lowercase value.

```
element.nodeName.toLowerCase() === 'input' ? element.value : element.innerText,
```

Obviously this should be changed everywhere, but I'm only making this comment.

https://stackoverflow.com/questions/254302/how-can-i-determine-the-type-of-an-html-element-in-javascript#254313
https://developer.mozilla.org/en-US/docs/Web/API/Element/localName
https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName
https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeName